ecosystem/sep-0034.md

```diff
@@ -0,0 +1,84 @@
+## Preamble
+
+```
+SEP: To Be Assigned
+Title: Client Application Attribution
+Author: Jake Urban, Leigh McCulloch
+Track: Standard
+Status: Draft
+Created: 2020-08-17
+Discussion: https://github.com/stellar/stellar-protocol/issues/685
+```
+
+## Simple Summary
+This protocol outlines how a client can prove to a web server that the application requesting the relevant operation is owned by the client.
+
+## Motivation
+In general, web services may want to attribute incoming requests to client applications on a per-operation basis. Because requests headers like `User-Agent` can be spoofed, anchors require a cryptographically secure method of verifying a client's identity.
+
+For example, an anchor server may not charge fees for transactions created using a particular wallet. In this case, the operation is processing a deposit or withdrawal transaction, but the operation could be any action related to different use cases.
+
+## Abstract
+This following protocol involves 3 parties:
+
+- the client application: the party requesting an operation from the attribution server 
+- the client's signing server: the web server that claims ownership of the client application
```

Comment:
I think that Morley's question is indicative that it's not very clear that `the client application` and `the client's signing server` are both operated by the same entity. 
Maybe `Wallet` and `Wallet Backend`?